src/main/resources/lang/en_US.toml

```diff
@@ -26,18 +26,24 @@ examples = "**Examples**:"
     last = "Last Airing"
     next = "Next Airing"
     age = "Age Rating"
+    description = "Search for anime information"
     # Color
     [commands.fun.color]
     error = "Error writing color"
     notcolor = "That's not a color"
+    description = "Show the color of a informed hex code"
     # Flip
     [commands.fun.flip]
     heads = "flipped a coin and got **Heads**."
     tails = "flipped a coin and got **Tails**."
+    description = "Flip's a coin"
     # Roll
     [commands.fun.roll]
     roll = "rolled"
     nanumber = "not a number"
+    description = "Roll's a random number"
+    [commands.fun.quote]
+    description = "Quote is a simple command which \"generate\" a random quote every time that is called."
```

Comment:
`Quote is a simple command which generates a random quote every time that it is called.`